tests/installation/release_notes_from_url.pm

```diff
@@ -10,13 +10,20 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 use testapi;
-use Utils::Backends;
-use serial_terminal 'select_serial_terminal';
+# use Utils::Backends;
+# use serial_terminal 'select_serial_terminal';
```

Comment:
sorry I merged accidentally with this commented code, could you please address it?